sa/db-next/boulder_sa/20240514000000_Paused.sql

```diff
@@ -0,0 +1,20 @@
+-- +migrate Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+-- This table has no auto-incrementing primary key because we don't plan to
+-- partition it. This table expected to be < 800K rows initially and grow at a
+-- rate of ~18% per year.
+
+CREATE TABLE `paused` (
+  `registrationID` bigint(20) NOT NULL,
+  `identifierType` tinyint(4) NOT NULL,
+  `identifierValue` varchar(255) NOT NULL,
+  `pausedAt` datetime DEFAULT NULL,
```

Comment:
It looks like you're moving forward with the "don't null-out the pausedAt column" approach we discussed, so this can become NOT NULL in the schema.